ilc/templates/error.ejs

```diff
@@ -0,0 +1,396 @@
+<!DOCTYPE html>
```

Comment:
_For next PR_
I came up with a great idea how we can store this file in a better way:
Add it to the registry as "error template" and fetch at ILC startup to build "pre-heated cache".